examples/fields.go

```diff
@@ -45,7 +45,7 @@ func main() {
 	// create volume
 	volume, err := volSvc.CreateVolume(newVolume)
 	if err != nil {
-		fmt.Println("Failed to create volume, ", err)
+		fmt.Println("Failed to create volume, err: %v", err)
```

Comment:
I think this one was correct before.  You might want to use Printf with this change.

Check out the Go Playground for an easy way to experiment with how the print functions work: https://play.golang.org/.

Reply:
ah ok I didn't notice some of them were println and the others were printf